docs/ops/normalization/MVN_6.md

```diff
@@ -0,0 +1,95 @@
+## MVN <a name="MVN"></a> {#openvino_docs_ops_normalization_MVN_6}
+
+**Versioned name**: *MVN-6*
+
+**Category**: *Normalization*
+
+**Short description**: Calculates mean-variance normalization of the input tensor.
+
+**Detailed description**
+
+*MVN* subtracts mean value from the input blob:
+\f[
+o_{i} = i_{i} - ReduceMean(i_{k}, axes)
+\f]
+If *normalize_variance* is set to `true`, the output blob is divided by variance. In the case of `eps_mode` equal to `inside_sqrt`:
```

Comment:
Consider changing 
``In the case of `eps_mode` equal to `inside_sqrt`:`` -->
``If `eps_mode` is equal to `inside_sqrt`:``